db/migrate/20171011140150_create_medicaid_applications.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMedicaidApplications < ActiveRecord::Migration[5.1]
+  def change
+    create_table :medicaid_applications do |t|
+      t.boolean :michigan_resident, null: false
+
+      t.timestamps
```

Comment:
super minor: I prefer timestamps first -- but this will be irrelevant in schema when you merge your other PR :)

Reply:
Rebased it onto that merged branch and re-ran the schema dump so this should be settled up :)